README.md

```diff
@@ -29,6 +29,10 @@ export KUBECONFIG=</path/to/kubeconfig>
 
 Build the docker image to run the placement controller.
 ```sh
+mkdir -p $GOPATH/src/github.com/open-cluster-management-io
+cd $GOPATH/src/github.com/open-cluster-management-io
+git clone https://github.com/open-cluster-management-io/placement.git
+cd placement
```

Comment:
@kim-fitness I tested in my environment. The image can be built without those steps. Why do you think they are mandatory?

By the way, in this readme, we assume that user has already cloned this repo to local disk, and the current working dir is the directory of the repo.

Reply:
Anyway, I think it's fine to add one step before `Deploy the placement controller`, like

### Clone this repo
```sh
git clone https://github.com/open-cluster-management-io/placement.git
cd placement
```

### Deploy the placement controller
... ...

Reply:
Hi @elgnay ,
Thanks for the feedback. It is OK to remove the following two steps from the README file and make the image building success. Let me make a new commit to this PR. 
```
mkdir -p $GOPATH/src/github.com/open-cluster-management-io
cd $GOPATH/src/github.com/open-cluster-management-io
```